cmd/yamlfmt/yamlfmt_test.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestFetchYaml(t *testing.T) {
+	sourceYaml := ` # See the OWNERS docs at https://go.k8s.io/owners
+approvers:
```

Comment:
Are there specific deficiencies you are testing here?  Maybe comment (e.g. "this proves it formats indents") and/or have differetn test cases for each flavor of dificiecency?

Reply:
@thockin essentially i want all the yamls to be as uniform as possible. So that when i write some code to parse and edit the yaml, the output will be as close to what's checked into k/k repository.

example, i want to cleanup the OWNERS files and i have some code that uses yaml.v3 to do this https://github.com/dims/maintainers/blob/main/yaml_utils.go it ends up with a lot of diffs if we don't have uniformity, so i think of this as a c beautifier, gofmt etc where you run the tool to ensure the file looks a certain way. 

With this PR, we standardize how the OWNERS yaml looks, so the tools that manipulate the yaml do NOT generate a lot of unnecessary diff(s).